src/loaders/compute_tools/microtrait/microtrait_util.py

```diff
@@ -0,0 +1,68 @@
+def decode_rule_boolean(
```

Comment:
Oh, I just found out there is this ruleunwrapped table. We probably don't need to do this decode ourselves. However, I don't remember anyone mentioning the existence of such a table.
https://github.com/ukaraoz/microtrait/blob/master/data-raw/microtrait_ruleunwrapped.txt

Reply:
Ulas just pointed to the `data-raw` directory, so I think anything in there is fair game if it makes things simpler. If you can use that table to simplify the function I'd do it